### PR TITLE
PKCS#8 DER Compatibility

### PR DESCRIPTION
**Pull request by chipx0r**
Demo Key: https://github.com/chipx0r/crackpkcs8/blob/master/AAA010101AAA.key
Pass: 12345678a

My CA gives us DER encoded certificates and I saw the chance to modify your program, in the reading part to be able to use it. I'm not an expert in python, but it worked.

### CODE REVIEW

File: run/pem2john.py [hunk 1]
```diff
@@ -40,57 +40,85 @@
    70:d=1  hl=4 l= 640 prim: OCTET STRING      [HEX DUMP]:C4BC6BC5447BED58...
 """
 
+def is_binary(filename):
```
Comment (kholia) [inline run/pem2john.py]: Is this really necessary?

I will try to come up with an alternate approach.

Comment (chipx0r) [reply run/pem2john.py]: *handled it

Comment (chipx0r) [reply run/pem2john.py]: > 
> 
> Is this really necessary?
> 
> I will try to come up with an alternate approach.

Would it be OK for you if I made the changes as I suggest with an argv[] param?

**Comment by magnumripper** [comment]
@kholia I'm totaly fine with you needing more time, just say so. Or should we merge?

**Comment by magnumripper** [comment]
@kholia I'm merging this now. You can tweak it later if needed!

**Pull request merged**